### ISSUE TITLE
Accept all button leads to exception only once

### ISSUE THREAD

**Issue by dpliskov** [SourcePointUSA/android-cmp-app#236]
**Describe the bug**
If tap Zustimmen (Accept All) button when Internet is disabled we get an exception only once - after first tap

**About your campaign (please complete the following information):**
 - Account Id = 75
 - Property Id = 4600
 - Property Name = WELT-NEWS-ANDROID
 - PM_ID = 134094

** About your environment **
 - SDK version= 5.3.3
 - Device: Samsung Galaxy S10
 - OS: Android 9

**To Reproduce**
1. Open Native Consent Screen
2. Disable Internet
3. Tap on Accept All button
4. Exception comes to onError callback
5. Tap on Accept All button
6. Exception doesn't come to onError callback

**Expected behavior**
Exception should come to onError callback every time when tap Accept All button

**Comment by andresilveirah** [comment]
Thanks for reporting @dpliskov . We'll try to reproduce this on our side and get back to you.

**Comment by leodoin** [comment]
@dpliskov , this is actually the expected behaviour: the `onError(ConsentLibException)` will be called only once by the SDK and it is up to you to close/detach the consent view. Does it make sense to you?

**Comment by dpliskov** [comment]
What if i need to make a possibility to retry this action. I will not get any answer after this retry attempts. I need to close native screen and show it again.

**Comment by leodoin** [comment]
> What if i need to make a possibility to retry this action without closing native screen. I will not get any answer after this retry attempts. I have to close native screen and show it again.

Precisely. Most errors are not recoverable, so the SDK forces you (by design) the reinstantiate it and ask for the msg content again. Long story short: if any error occur the SDK will be finished (no more callbacks) and you should close all views that you may have on your tree (for this matter you can profit from the `onConsentUIFinished` callback)

**Comment by leodoin** [comment]
@dpliskov, pls let us know if the above approach suites you, so we can't hopefully close this issue :)
